runtime/calamari/src/lib.rs

```diff
@@ -131,70 +131,91 @@ pub const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(70);
+    pub RuntimeBlockWeights: BlockWeights = BlockWeights::builder()
+        .base_block(BlockExecutionWeight::get())
+        .for_class(DispatchClass::all(), |weights| {
+            weights.base_extrinsic = ExtrinsicBaseWeight::get();
+        })
+        .for_class(DispatchClass::Normal, |weights| {
+            weights.max_total = Some(NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT);
+        })
+        .for_class(DispatchClass::Operational, |weights| {
+            weights.max_total = Some(MAXIMUM_BLOCK_WEIGHT);
+            // Operational transactions have some extra reserved space, so that they
+            // are included even if block reached `MAXIMUM_BLOCK_WEIGHT`.
+            weights.reserved = Some(
+                MAXIMUM_BLOCK_WEIGHT - NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT
+            );
+        })
+        .avg_block_initialization(AVERAGE_ON_INITIALIZE_RATIO)
+        .build_or_panic();
+    pub const SS58Prefix: u8 = manta_primitives::constants::CALAMARI_SS58PREFIX;
 }
 
 impl pallet_tx_pause::Config for Runtime {
-	type Event = Event;
-	type UpdateOrigin = EnsureOneOf<
-		EnsureRoot<AccountId>,
-		pallet_collective::EnsureProportionMoreThan<AccountId, TechnicalCollective, 1, 2>,
-	>;
-	type WeightInfo = weights::pallet_tx_pause::SubstrateWeight<Runtime>;
+    type Event = Event;
+    type UpdateOrigin = EnsureOneOf<
+        EnsureRoot<AccountId>,
+        pallet_collective::EnsureProportionMoreThan<AccountId, TechnicalCollective, 1, 2>,
+    >;
+    type WeightInfo = weights::pallet_tx_pause::SubstrateWeight<Runtime>;
 }
 
 // Don't allow permission-less asset creation.
 pub struct BaseFilter;
 impl Contains<Call> for BaseFilter {
-	fn contains(call: &Call) -> bool {
-		if matches!(
-			call,
-			Call::Timestamp(_) | Call::ParachainSystem(_) | Call::System(_)
-		) {
-			// always allow core call
-			// pallet-timestamp and parachainSystem could not be filtered because
-			// they are used in communication between relaychain and parachain.
-			return true;
-		}
-
-		if pallet_tx_pause::PausedTransactionFilter::<Runtime>::contains(call) {
-			// no paused call
-			return false;
-		}
-
-		match call {
-			| Call::Authorship(_)
-			| Call::Multisig(_)
+    fn contains(call: &Call) -> bool {
+        if matches!(
+            call,
+            Call::Timestamp(_) | Call::ParachainSystem(_) | Call::System(_)
+        ) {
+            // always allow core call
+            // pallet-timestamp and parachainSystem could not be filtered because
+            // they are used in communication between relaychain and parachain.
+            return true;
+        }
+
+        if pallet_tx_pause::PausedTransactionFilter::<Runtime>::contains(call) {
+            // no paused call
+            return false;
+        }
+
+        #[allow(clippy::match_like_matches_macro)]
+        // keep CallFilter with explicit true/false for documentation
+        match call {
+            // Explicitly DISALLOWED calls
```

Comment:
ok i concede that they do look wrong in github. they're not indented as far as they should be.
vscode looks normal though. not sure which display source is wrong here

Reply:
it looks like rustfmt did *NOT* automatically convert all leading tabs to spaces.
I ran a simple replace of tabs -> 4 spaces to normalize this.
I hope rustfmt will at least enforce automatically that no new tabs are added to the sources.

Reply:
I tried adding a new change with leading tab instead of spaces. `cargo fmt --all -- --check` fails as expected. Should be good for CI use then.
We're guaranteed nobody introduces new tabs into the codebase.
cargo fmt checks .rs files and taplo checks .tomls